applications/common/server/Dockerfile

```diff
@@ -1,24 +1,21 @@
 ARG REGISTRY
 ARG TAG=latest
-FROM ${REGISTRY}cloudharness-base:${TAG}
+FROM ${REGISTRY}cloudharness-flask:${TAG}
+
+ENV MODULE_NAME=common
+
+ENV WORKERS=2
+ENV PORT=8080
 
 RUN apk update && apk add postgresql-dev gcc python3-dev musl-dev
```

Comment:
musl-dev, python3-dev, gcc can be removed, they are part of the cloudharness-base image